cloudigrade/internal/tests/views/test_dailyconcurrentusageviewset.py

```diff
@@ -0,0 +1,312 @@
+"""Collection of tests for AccountViewSet."""
+import datetime
+from unittest.mock import patch
+
+import faker
+from django.test import TransactionTestCase
+from django.utils.translation import gettext as _
+from rest_framework.test import APIClient, APIRequestFactory
+
+from api.models import ConcurrentUsageCalculationTask
+from api.tests import helper as api_helper
+from util.misc import get_today, get_yesterday
+from util.tests import helper as util_helper
+
+
+class InternalDailyConcurrentUsageViewSetTest(TransactionTestCase):
```

Comment:
Like with `DailyConcurrentUsageViewSet` and `InternalDailyConcurrentUsageViewSet`, this looks like a copy-paste job. If we change the the viewset classes so that one extends the other, we don't need to duplicate all these tests. We can simply have the tests for the subclass check only the things that are different in the subclass.